frontend/catalyst/api_extensions/error_mitigation.py

```diff
@@ -46,7 +53,7 @@ def mitigate_with_zne(fn=None, *, scale_factors=None, deg=None):
     Args:
         fn (qml.QNode): the circuit to be mitigated.
         scale_factors (array[int]): the range of noise scale factors used.
-        deg (int): the degree of the polymonial used for fitting.
+        extrapoloate (Callable): A function to perform fitting.
```

Comment:
typo: s/extrapoloate/extrapolate

Reply:
It's not clear how the `extrapolate` Callable should look like. A more explicit comment would help here, since this is exposed at the level of API.